lib/FeedItem.php

```diff
@@ -428,6 +428,37 @@ public function setUid($uid) {
 		return $this;
 	}
 
+	/**
+	 * Get duration (for items where the link points to media)
+	 *
+	 * Use {@see FeedItem::setDuration()} to set the duration.
+	 *
+	 * @return int Duration in seconds.
+	 */
+	public function getDuration() {
+		return $this->duration;
```

Comment:
Please also add a corresponding property to the class – not doing so will cause issues with newer PHP versions: https://wiki.php.net/rfc/deprecate_dynamic_properties

Reply:
Looks like it needs to be added to [few more places](https://github.com/RSS-Bridge/rss-bridge/blob/8f0d90f653702f4a38e1069f655d140b0faa1cce/lib/FeedItem.php#L456-L524):

- `__get`
- `__set`
- `toArray`